features/graphql/mutation.feature

```diff
@@ -126,7 +126,7 @@ Feature: GraphQL mutation support
     When I send the following GraphQL request:
     """
     mutation {
-      createRelationEmbedder(input: {paris: "paris", krondstadt: "Krondstadt", anotherRelated: {id: 2, symfony: "laravel"}, clientMutationId: "myId"}) {
+      createRelationEmbedder(input: {paris: "paris", krondstadt: "Krondstadt", anotherRelated: {_id: "/related_dummies/2", symfony: "laravel"}, clientMutationId: "myId"}) {
```

Comment:
Looks invalid to me. `_id` refer to the editable "id" property (so, not an IRI, something like an UUID or a random string), `id` on the other hand is the "obfuscated" global ID defined by the relay spec.

Here it can be:

* `_id: 2`
* or `id: "/related_dummies/2"` here

But not this one

Reply:
Yes that's what I said in the PR's description.